view/js/main.js

```diff
@@ -395,6 +395,19 @@ function NavUpdate() {
 	timer = setTimeout(NavUpdate, updateInterval);
 }
 
+function callAddonHooks(typeOfHook) {
```

Comment:
Can you please move this function declaration to `addon-hooks.js`?

Additionally, can you please add a comment block to explain what this function does?